tempesta_fw/work_queue.h

```diff
@@ -38,13 +38,37 @@ typedef struct {
 	long			last_head;
 	atomic64_t		head ____cacheline_aligned;
 	atomic64_t		tail ____cacheline_aligned;
+	unsigned long		flags;
 } TfwRBQueue;
 
+enum {
+	/* Enable IPI generation. */
+	TFW_QUEUE_B_IPI = 0
```

Comment:
Probably we don't need `_B_` suffix meaning `bit` here. For other flags we historically used `_B_` and `_F_` suffixes to distinguish between flags and bits, but now we tend to use common bitmap API instead of 'manual' bit operations. Removing of `_B_` suffix will synchronise the code with upcoming patches.

Reply:
Corrected.